src/pages/Verification/Verification.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState, useEffect } from "react";
+import { Typography, Container, Box, Button } from "@mui/material";
+import { MailOutline } from '@mui/icons-material';
+import { selectToken, selectUser } from "@/redux/userSlice";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { RootState } from '@/redux/store';
+import { BASE_URL } from '@/config/default';
+import axios, { AxiosError, AxiosResponse } from "axios";
+import AlertMessage from "@/components/ui/AlertMessage";
+
+interface AuthResponse {
```

Comment:
No need to redefine this type here. I have similar type defined in `src/types/AuthTypes.ts`